### ISSUE TITLE
[build] Enable KVM in QEMU

### ISSUE THREAD

**Issue by rafael0121** [nanvix/kernel#422]
## Description
Enable KVM in QEMU allow Nanvix run on a more realistic environment.

### Change:

* `scripts/run.sh`

## QEMU Options
To enable KVM is necessary use new options:
- enable-kvm -> Enable KVM virtualization.
- cpu host -> Pass the host CPU model and feature to the guest.

**Comment by ppenna** [comment]
@rafael0121 as far as I understand enabling KVM is orthogonal to supporting IOMMU.

For instance, QEMU allows you supply the `- - enable-kvm`  option for the PC and ISAPC machines.

Could you clarify why IOMMU is required, given the current features that Nanvix has?

**Comment by rafael0121** [comment]
> @rafael0121 as far as I understand enabling KVM is orthogonal to supporting IOMMU.
> 
> For instance, QEMU allows you supply the ` enable-kvm` option for the PC and ISAPC machines.
> 
> Could you clarify why IOMMU is required, given the current features that Nanvix has?

IOMMU allows we assign PCI devices directly for VM, i think it isn't necessary for while, but maybe support this now would help in future.

**Comment by ppenna** [comment]
> > @rafael0121 as far as I understand enabling KVM is orthogonal to supporting IOMMU.
> > For instance, QEMU allows you supply the ` enable-kvm` option for the PC and ISAPC machines.
> > Could you clarify why IOMMU is required, given the current features that Nanvix has?
> 
> IOMMU allows we assign PCI devices directly for VM, i think it isn't necessary for while, but maybe support this now would help in future.

Yes, that is what IOMMU is all about, and we eventually want to support it.

Anyways this is orthogonal to supporting KVM. So let's just enable kvm on supported targets.
